SP-15 added tabbarcontroller

eunery opened the pull request.

eunery committed:
SP-10 setup project

https://trello.com/c/RXDpktnY/10-подготовка

eunery committed:
SP-15 added tabbarcontroller

https://trello.com/c/RL8tRok4/15-ios-реализация-maintabbarcontrollerглавного-экрана

dkniss commented on ios-online-cinema.xcodeproj/project.pbxproj:
```diff
@@ -684,6 +700,7 @@
 				INFOPLIST_FILE = "ios-online-cinema/Info.plist";
 				INFOPLIST_KEY_UIApplicationSupportsIndirectInputEvents = YES;
 				INFOPLIST_KEY_UILaunchStoryboardName = LaunchScreen;
+				INFOPLIST_KEY_UIStatusBarStyle = "";
```
Зачем добавился этот ключ?

dkniss commented on ios-online-cinema/Coordinators/MoviesFeedCoordinator.swift:
```diff
@@ -0,0 +1,38 @@
+    
+    init(_ navVC: UINavigationController) {
+        self.navigationController = navVC
+        
```
лишний пробел

dkniss commented on ios-online-cinema/Coordinators/Protocols/TabBarPage.swift:
```diff
@@ -0,0 +1,56 @@
+    func pageIcon() -> UIImage {
+        switch self {
+        case .moviesFeed:
+            return UIImage(named: "feed")!
```
force unwrap

dkniss commented on ios-online-cinema/Coordinators/Protocols/TabBarPage.swift:
```diff
@@ -0,0 +1,56 @@
+        case .moviesFeed:
+            return UIImage(named: "feed")!
+        case .favouriteMovies:
+            return UIImage(named: "like")!
```
force unwrap

dkniss commented on ios-online-cinema/Coordinators/Protocols/TabBarPage.swift:
```diff
@@ -0,0 +1,56 @@
+    
+    func pageIconSelected() {
+        
+    }
```
Для чего нужен этот метод? Если не нужен, то удаляем

dkniss commented on ios-online-cinema/Coordinators/TabBarCoordinator.swift:
```diff
@@ -0,0 +1,127 @@
+    
+    required init(_ navigationController: UINavigationController) {
+        self.navigationController = navigationController
+//        self.tabBarController = .init()
```
Удалить коммент

dkniss commented on ios-online-cinema/Coordinators/TabBarCoordinator.swift:
```diff
@@ -0,0 +1,127 @@
+    
+    
+    func start() {
+        print("TabBarCoordinator starts!")
```
удаляй принты, понимаю что их использовал для дебага, но в коммит они не должны попадать

dkniss commented on ios-online-cinema/Coordinators/TabBarCoordinator.swift:
```diff
@@ -0,0 +1,127 @@
+    }
+    
+    deinit {
+        print("TabBarCoordinator deinit")
```
удалить принт, да и вообще деинит не нужен тогда будет его тоже можешь убрать

dkniss commented on ios-online-cinema/Coordinators/TabBarCoordinator.swift:
```diff
@@ -0,0 +1,127 @@
+    }
+    
+    
+    
```
2 лишних пробела

dkniss commented on ios-online-cinema/Coordinators/FavouriteMoviesCoordinator.swift:
```diff
@@ -0,0 +1,36 @@
+    }
+    
+    func start() {
+        print("FavouriteMoviesCoordinator starts!")
```
убрать принт

dkniss commented on ios-online-cinema/Coordinators/MoviesFeedCoordinator.swift:
```diff
@@ -0,0 +1,38 @@
+    }
+    
+    func start() {
+        print("MoviesFeedCordinator starts!")
```
убрать принт

eunery committed:
SP-15 fix issues

https://trello.com/c/RL8tRok4/15-ios-реализация-maintabbarcontrollerглавного-экрана

eunery committed:
Revert "SP-15 fix issues"

This reverts commit 61d1ab923e3ea495c0a6adc7a5d3c5161eff72fa.

eunery committed:
SP-15 fix coordinator

https://trello.com/c/RL8tRok4/15-ios-реализация-maintabbarcontrollerглавного-экрана
